### DISCUSSION TITLE
[공지]❗❗수정 항목 ❗❗프로젝트 마무리 전 체크하기

### DISCUSSION THREAD

**Discussion by jny4867** [prgrms-fe-devcourse/NFE1_2_2_MUX#115] [A. 업데이트 및 공지]
지금 시점에서 수정이 필요한 부분입니다!
참고하여 작업 후 수정 부탁드립니다 :)

mainpage 

- [x] 글귀 수정
- [ ] 반응형 구현
- [ ] 포스트 섹션에서 음원까지 가져옴
- [ ] 포스트섹션 포스트 누르면 네모네모빔
- [ ] 프로필 간격 줄이기
- [x] 글귀 크기 좀 줄이기

Nav 

- [x] 칭호 없애기
- [x] 검색창 좀 왼쪽으로
- [x] 버튼 간격 넓히기
- [x] 프로필 포인트 커서

login

- [x] 로그인에 실패했습니다 문구 띄우기

큐레이션 아티스트

- [x] 호버하면 재생 버튼 뜨도록 변경 (현재는 계속 떠 있음)
- [x] 큐레이션 트랙 문구 변경

프로필페이지 

- [x] 포스트 가로 스크롤
- [x] 음원 카드 컴포넌트 변경
- [x] 가운데 정렬
- [x] 버튼 바이오 밑으로 내리기
- [x] 버튼이 초기에 바로 안 뜸

랜딩

- [x] 이미지 바꾸기

**Comment by hjlee2778** [comment]
1시 52분 기준

mainpage 
- [ ] 글귀 수정 
- [ ] 반응형 구현
- [ ] 포스트 섹션에서 음원까지 가져옴
- [ ] 포스트섹션 포스트 누르면 네모네모빔

Nav 

- [x] 칭호 없애기
- [x] 검색창 좀 왼쪽으로
- [x] 버튼 간격 넓히기
- [x] 프로필 포인트 커서
- [x] 반응형

login
- [x] 라우팅 설정
- [x] 로그인에 실패했습니다 문구 띄우기

큐레이션 아티스트

- [x] 호버하면 재생 버튼 뜨도록 변경 (현재는 계속 떠 있음)
- [x] 큐레이션 트랙 문구 변경

프로필페이지 

- [x] 포스트 가로 스크롤
- [x] 음원 카드 컴포넌트 변경
- [x] 가운데 정렬
- [x] 버튼 바이오 밑으로 내리기

랜딩

- [x] 이미지 바꾸기
